utam-compiler/src/main/java/utam/compiler/translator/InterfaceSerializer.java

```diff
@@ -53,6 +54,12 @@ private String getPackageName() {
   }
 
   private void addMethodDeclaration(List<String> out, MethodDeclaration declaration) {
+    // The interface extends the PageObject interface, which already has a load
+    // method, so we don't need to add a load method to the generated interface.
+    if (declaration.getName().equals(BEFORE_LOAD_METHOD_NAME)) {
```

Comment:
This should probably verify that `source.getBaseInterfaceType().getSimpleName()` is the interface the `PageObject` interface too.

Reply:
It _could_ verify that, but the verification would be superfluous, as the only thing this class will ever serialize is an interface extending `PageObject`. If that ever changes, yes, we'd probably want to make a change here.